Pa 82 mindmap/todo implement completed tasks UI

usefSaeed opened the pull request:
Completed tasks UI fully done
It is as follows:
1. Induced in the TODO page
2. Shown/hidden using switch in todo component
3. Has dummy data temporarly awaiting integration
4. Buttons effect won't work before integration

NB: I am ready to help @NoirFLamme in #PA-77 Integrate Completed Tasks if needed

NoirFLamme reviewed (approved):
Good job

AliELSharawy reviewed (changes requested):
Great Job :)
but remove comments and try the whole application.

usefSaeed commented:
Changes made:
1. Uncommented other pages in navigation
2. Removed some task data from Completed Component as agreed upon in the design
Data removed:
- whether task is repeated
-task due date
3. Decreased the width of the completed component in large screens so that todolist would take a larger area of the screen
4. Added TODO comments for API methods that related yo task #PA-77

NB: The viewTaskDetails method awaits the AddTask Component
